recipes-kernel/linux/5.15/patches/xen-txt-add-xen-txt-eventlog-module.patch

```diff
@@ -277,7 +277,7 @@ Subject: [PATCH] xen-txt: add xen txt eventlog module
  #define __HYPERVISOR_xc_reserved_op       39 /* reserved for XenClient */
  #define __HYPERVISOR_xenpmu_op            40
  #define __HYPERVISOR_dm_op                41
-+#define __HYPERVISOR_txt_op               42
++#define __HYPERVISOR_txt_op               43
```

Comment:
Is there no patch fuzz associated with this? Shouldn't there be a line for the new hypfs?

Reply:
No, log.do_patch doesn't mention any.  This is patching a header in linux source, and that has not been updated with hypfs_op yet.